.goreleaser.yml

```diff
@@ -1,22 +1,37 @@
 builds:
-  - main: ./cmd/gotk
-    ldflags:
-      - -s -w -X main.VERSION={{ .Version }}
-    binary: gotk
+  - <<: &build_defaults
+      binary: gotk
+      main: ./cmd/gotk
+      ldflags:
+        - -s -w -X main.VERSION={{ .Version }}
+      env:
+        - CGO_ENABLED=0
+    id: linux
     goos:
-      - darwin
       - linux
-      - windows
     goarch:
       - amd64
       - arm64
       - arm
-    goarm:
-      - 7
```

Comment:
Removing this produces v6 binaries and that breaks ARMv7 compatibility.

Reply:
my bad. it's a mistake.